src/components/DataView/DataView.tsx

```diff
@@ -27,6 +27,15 @@ const StyledWrapper = styled.div`
 		margin-bottom: 8px;
 	}
 
+	& > .headerList {
+		display: flex;
+		flex-direction: column;
+	}
+
+	& > .headerRow.title {
+		margin-left: 12px;

```

Comment:
Looking at the figma I couldn't find these 12px. The title bar should have 24px all around except at the bottom (it should be 32) but you already added those in the other ticket.